CRVResponse/fcl/prolog_v11.fcl

```diff
@@ -9,7 +9,9 @@
 #include "Offline/CommonMC/fcl/prolog.fcl"
 BEGIN_PROLOG
 
-    DigitizationStart: 400.0 //ns after event window start (i.e. 400ns after first clock tick after POT) --> 400ns...425ns after POT
+    DigitizationStart :  200.0 //ns after event window start (400ns...425ns after POT)
```

Comment:
Is this number specific to the CRV?

Reply:
It's not set in stone yet, but we wanted to start somewhere around 400ns after POT.